### PR TITLE
0.2.0 release

### LINKED ISSUE #619: JanusGraph 0.2.0 Release
Finalize documentation and version for 0.2.0 release

### PR DESCRIPTION
**Pull request by pluradj**
Fixes #619 

Thank you for contributing to JanusGraph.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there an issue associated with this PR? Is it referenced in the commit message?
- [x] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [x] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [x] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: CHANGELOG.asc [hunk 1]
```diff
@@ -2,6 +2,31 @@
...
+* Apache HBase 0.98.24-hadoop2, 1.2.6
+* Google Bigtable 1.0.0-pre3
+* Oracle BerkeleyJE 7.3.7
+* Elasticsearch 5.6.2
+* Apache Lucene 7.0.0
+* Apache Solr 7.0.0
+* Apache TinkerPop 3.2.6
+* Java 1.8
+
+For more information on features and bug fixes in 0.2.0, see these GitHub milestone:
```
Comment (twilmes) [inline CHANGELOG.asc]: Looks like I had typo in the 0.1.1 notes that got copied into here: `see these GitHub` -> `see the GitHub`

File: docs/changelog.txt [hunk 2]
```diff
@@ -2,6 +2,31 @@
...
+* Apache HBase 0.98.24-hadoop2, 1.2.6
+* Google Bigtable 1.0.0-pre3
+* Oracle BerkeleyJE 7.3.7
+* Elasticsearch 5.6.2
+* Apache Lucene 7.0.0
+* Apache Solr 7.0.0
+* Apache TinkerPop 3.2.6
+* Java 1.8
+
+For more information on features and bug fixes in 0.2.0, see these GitHub milestone:
```
Comment (twilmes) [inline docs/changelog.txt]: Same typo from above: s/these/the.

File: pom.xml [hunk 3]
```diff
@@ -60,7 +60,7 @@
         <connection>scm:git:git@github.com:JanusGraph/janusgraph.git</connection>
         <developerConnection>scm:git:git@github.com:JanusGraph/janusgraph.git</developerConnection>
         <url>git@github.com:JanusGraph/janusgraph.git</url>
-        <tag>HEAD</tag>
+        <tag>0.2.0</tag>
     </scm>
     <properties>
         <janusgraph.compatible.versions>0.1.0,0.1.1</janusgraph.compatible.versions>
```
Comment (amcp) [inline pom.xml]: does 0.2.0 need to go here too?

Comment (pluradj) [reply pom.xml]: no, `janusgraph.compatible.versions` is used for backwards compatibility with previous releases.

File: CHANGELOG.asc [hunk 4]
```diff
@@ -2,6 +2,31 @@
...
+<dependency>
+   <groupId>org.janusgraph</groupId>
+   <artifactId>janusgraph-core</artifactId>
+   <version>0.2.0</version>
+</dependency>
+
+*Tested Compatibility:*
+
+* Apache Cassandra 2.1.18, 2.2.10, 3.0.14, 3.11.0
+* Apache HBase 0.98.24-hadoop2, 1.2.6
```
Comment (sjudeng) [inline CHANGELOG.asc]: We also tested (at least default tests) for HBase 1.3.1

**Comment by amcp** [comment]
Is there an issue for 0.2.0 release we can refer to in this PR? otherwise, lgtm

File: CHANGELOG.asc [hunk 5]
```diff
@@ -2,6 +2,31 @@
...
+* Google Bigtable 1.0.0-pre3
+* Oracle BerkeleyJE 7.3.7
+* Elasticsearch 5.6.2
+* Apache Lucene 7.0.0
+* Apache Solr 7.0.0
```
Comment (sjudeng) [inline CHANGELOG.asc]: Latest version of several Elasticserach and Solr/Lucene are also tested in Travis.

Comment (pluradj) [reply CHANGELOG.asc]: can you list or point me at the specific versions that were tested?

Comment (sjudeng) [reply CHANGELOG.asc]: https://github.com/JanusGraph/janusgraph/blob/58485ec48af498bee5569dcd2942fe136d5b80a0/janusgraph-es/pom.xml#L14-L17 and https://github.com/JanusGraph/janusgraph/blob/1b96a7c3ec2646c88df36ab47eec717a6727d18b/janusgraph-solr/pom.xml#L14-L16

**Comment by pluradj** [comment]
Fixes #619

**Comment by robertdale** [comment]
+1

**Comment by kavink** [comment]
@pluradj Does this mean 0.2.0 is released ? because docs still point to 0.1.1.

Also does 0.2.0 support Cassandra cql instead of Thrift ?

**Comment by pluradj** [comment]
@kavink no, it is not officially released. Our [release policy](http://docs.janusgraph.org/latest/release-policy.html) requires a vote from committers to approve the release. Please follow the [voting thread](https://groups.google.com/d/msg/janusgraph-dev/1COIv0_2mMo/EA9RRrgpAgAJ) on janusgraph-dev, and feel free to test it out. JanusGraph 0.2.0 supports CQL or Thrift, your choice.

**Pull request merged**